### DISCUSSION TITLE
GitOpsCon: The future and how it will be a success

### DISCUSSION THREAD

**Discussion by niklasmtj** [open-gitops/project#201] [General]
At the last meeting on Wednesday 6 November we briefly discussed possible future GitOpsCon formats. With many GitOpsCons in the past, we have come to the point where we need to define what a "successful" GitOpsCon might look like in the future. So I want to start this discussion by brainstorming possible "success" metrics or values that we want to keep in mind to guide future events. 

This will help us to organise future events (e.g. schedule and topics), but also to show potential sponsors what they can expect from the event.

Since there are many different possible scenarios for future GitOpsCons, the metrics and values can be either broad / general or specific to a particular type of event. Whatever you prefer :)

Examples of possible scenarios that have been discussed at past events
- face-to-face, stand-alone event
- face-to-face, co-located event
- face-to-face, Kubecon track
- online / remote, standalone event
- ... add more 

I am really looking forward to hearing what you would like to see at future GitOpsCon events.

**Comment by roberthstrand** [comment]
I really like the co-located event, but like me and @christianh814 talked about during KubeCon, we should be co-located with the right type of event. KubeCon is a perfect match for the demographic that attends it, Open Source Summit has some overlap but not a perfect match. The main issue is that there are so many co-located events these days, that it is difficult to squeeze in GitOpsCon.

If we did online, I think we would do fairly well, but in my opinion that is the last resort. Most people would prefer to meet and greet, and it is easier to concentrate on the agenda when not joining in remotely.
